### DISCUSSION TITLE
[@types/tabulator-tables] - cell.getData() should return 'any' instead of '{}'

### DISCUSSION THREAD

**Discussion by atebeta** [DefinitelyTyped/DefinitelyTyped#67968] [Issues with a @types package]
export interface CellComponent {
        
        /** The getData function returns the data for the row that contains the cell. */
        getData: (transformType?: "data" | "download" | "clipboard") => any;
        
        }

[a comment by an automated account is not shown]

**Comment by peterblazejewicz** [comment]
fyi: edited the title to bring up DT owners

**Comment by Jojoshua** [comment]
There is already discussion on the PR https://github.com/DefinitelyTyped/DefinitelyTyped/pull/67969#issuecomment-1870188813
